lib/hushed/blackboard.rb

```diff
@@ -40,5 +42,40 @@ def build_document(type, contents)
       end
       namespace.const_get(type).new(:io => contents) if namespace
     end
+
+    def retrieve_latest(document_name)
+      blackboard = client.from_quiet_bucket
+      documents = inventory_summary_documents(blackboard, document_name)
```

Comment:
actually i think `documents_with_prefix makes` most sense since this just gets all documents with the prefix, and then the most recent one is picked. thanks for pointing that out, that was a sloppy refactor on my part